socket.yml

```diff
@@ -1,4 +1,6 @@
 name: google-geocoding
+description: Socket for geocoding Google API
```

Comment:
Actually in last version of the Syncano version and description should be stored only in `package.json`, please close this PR.